config/kubeconfig/garden.yaml

```diff
@@ -0,0 +1,29 @@
+gardens:
+  - identity: sap-landscape-live
+    kubeconfig: ~/.gardener-prow/kubeconfig/kubeconfig--gardener--prow-combined.yaml
+    context: garden-garden-ci
+    patterns:
+      - ^((live|sap-landscape-live)/)?shoot--(?P<project>[^.]+)--(?P<shoot>[^.]+)$
+      - (live\.(k8s\.ondemand\.com|gardener\.cloud\.sap))(/namespace/(?P<namespace>[^/]+))?(/shoots/(?P<shoot>[^/]+))?
+      - (?P<shoot>[^.]+)\.(?P<project>[^.]+)\.shoot\.live\.k8s-hana\.ondemand\.com
+    accessRestrictions:
```

Comment:
Let's drop this. This is only operations-related for internal service obligations.